serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -21,5 +21,10 @@ You can enable JSON Web Token (JWT) authentication for Knative services.
 Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
 ====
 
+[IMPORTANT]
+====
+You must set the annotation `sidecar.istio.io/rewriteAppHTTPProbers: "true"` in your Knative service as {ServerlessProductName} versions 1.14.0 and later use a HTTP probe as the readiness probe for Knative services by default.
```

Comment:
s/annotation `sidecar.istio.io/rewriteAppHTTPProbers: "true"/`sidecar.istio.io/rewriteAppHTTPProbers: "true" annotation
s/a HTTP/an HTTP

Reply:
Thank you! Updated.